src/mudlet.cpp

```diff
@@ -3261,6 +3261,10 @@ QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1,
         // Takes two extra arguments (profile name, dataTime stamp) that returns
         // the pathFile name for a dateTime stamped map file:
         return QStringLiteral("%1/.config/mudlet/profiles/%2/map/%3map.dat").arg(QDir::homePath(), extra1, extra2);
+    case profileDateTimeStampedJsonMapPathFileName:
```

Comment:
Given that we have made this the same as the output from `profileDateTimeStampedMapPathFileName` do we really need it?

Reply:
Mm, one is `.json` and the other is `.dat`. That said the `profileDateTimeStampedMapPathFileName` name is most the most pleasant one, could use an improvement for developer aesthetics.

Reply:
> Mm, one is .json and the other is .dat....

:man_facepalming: 

Oh, silly me - didn't spot that - so not an issue after all.